### PR TITLE
[FIX] account_edi: We should delete the last info/warning/error msg w…

### PR DESCRIPTION
**Pull request by jco-odoo**
…hen success to not have something weird still in the view

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
Sometimes the last state of an edi document is not properly cleaned when success True.

File: addons/account_edi/models/account_edi_document.py [hunk 1]
```diff
@@ -127,7 +127,10 @@ def _postprocess_post_edi_results(documents, edi_result):
                     if not old_attachment.res_model or not old_attachment.res_id:
                         attachments_to_unlink |= old_attachment
                 if move_result.get('success') is True:
-                    document.state = 'sent'
+                    document.write({'state': 'sent',
+                                    'error': '',
+                                    'blocking_level': ''})
```
Comment (william-andre) [inline addons/account_edi/models/account_edi_document.py:132]:
```suggestion
                    document.write({
                        'state': 'sent',
                        'error': False,
                        'blocking_level': False,
                    })
```

Keep the same style and value as in `_postprocess_cancel_edi_results` (copy paste is bad :grimacing:)
Also, why is the `attachment_id` set to `False` when cancelling?

**Comment by jco-odoo** [comment]
@william-andre Changes done

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**